Corrections.py

```diff
@@ -132,7 +136,7 @@ def getNormalisationCorrections(df, config, sample, ana_cache=None, return_varia
     df = df.Define('genWeightD', 'std::copysign<double>(1., genWeight)')
     scale = 'Central'
     df, tau_SF_branches = tau.getSF(df, config)
-    all_branches = [ pu_SF_branches, tau_SF_branches ]
+    all_branches = [ pu_SF_branches ]
```

Comment:
this is correct, but we want to have product of all tauID SF as a separate branch. Better to implement it here and return them inside `all_weights`